tsl/test/expected/continuous_aggs_ddl-11.out

```diff
@@ -943,3 +939,107 @@ SELECT chunk_table, ranges FROM chunk_relation_size('drop_chunks_table');
  _timescaledb_internal._hyper_12_19_chunk | {"[50,60)"}
 (2 rows)
 
+--test drop_chunks with cascade_to_materialization set to true (github 1644)
+-- This checks if chunks from mat. hypertable are actually dropped 
+-- and deletes data from chunks that cannot be dropped from that mat. hypertable.
+SELECT format('%s.%s', schema_name, table_name) AS drop_chunks_mat_tablen
+    FROM _timescaledb_catalog.hypertable, _timescaledb_catalog.continuous_agg
+    WHERE _timescaledb_catalog.continuous_agg.raw_hypertable_id = :drop_chunks_table_nid
+        AND _timescaledb_catalog.hypertable.id = _timescaledb_catalog.continuous_agg.mat_hypertable_id \gset
+SELECT drop_chunks(table_name => 'drop_chunks_table', older_than=>integer_now_test2() + 200, cascade_to_materializations => true);
+               drop_chunks                
+------------------------------------------
+ _timescaledb_internal._hyper_12_19_chunk
+ _timescaledb_internal._hyper_12_15_chunk
+(2 rows)
+
+SELECT count(c) FROM show_chunks('drop_chunks_table') AS c;
+ count 
+-------
+     0
+(1 row)
+
+SELECT count(c) FROM show_chunks(:'drop_chunks_mat_tablen') AS c;
+ count 
+-------
+     0
+(1 row)
+
+SELECT set_chunk_time_interval('drop_chunks_table', 10);
+ set_chunk_time_interval 
+-------------------------
+ 
+(1 row)
+
+SELECT set_chunk_time_interval(:'drop_chunks_mat_tablen', 20);
+ set_chunk_time_interval 
+-------------------------
+ 
+(1 row)
+
+INSERT INTO drop_chunks_table select generate_series(1,35), 100;
+update drop_chunks_table set data = 250 where time = 25;
+update drop_chunks_table set data = 290 where time = 29;
+REFRESH materialized view drop_chunks_view;
+INFO:  new materialization range not found for public.drop_chunks_table (time column time): not enough new data past completion threshold of 60 as of 35
```

Comment:
Honestly, I have no clue what these INFO messages mean. Looks more like DEBUG output to me. Realize it is probably not an issue with this PR, but I am still pointing this out because this must be quite confusing for users. I suggest creating an issue to clean up these messages.

As a user-facing message, I think this should be something more akin to:

`continuous aggregate for "public.drop_chunks_table" was not expanded due to lack of new data`

Perhaps more deeper technical information is suitable for a `errdetail` message.

Reply:
This command is more of a debugging command and used in our test cases as we don't want the bgw jobs to do the refresh.
It has been helpful to have these messages in the pg log files (for bgw job runs) for helping users who are unfamiliar with cont. aggregates or figuring out issues with bgw jobs.

Reply:
INFO is user-facing (sent to client) so I still find this odd even if we recommend bgw doring the refresh over a manual `REFRESH`.